src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -12088,6 +12088,6 @@ let TcAndPublishValSpec (cenv: cenv, env, containerInfo: ContainerInfo, declKind
         let xmlDoc = xmlDoc.ToXmlDoc(checkXmlDocs, paramNames)
         let vspec = MakeAndPublishVal cenv env (altActualParent, true, declKind, ValNotInRecScope, valscheme, attrs, xmlDoc, literalValue, false)
 
-        assert(vspec.InlineInfo = inlineFlag)
+        //assert(vspec.InlineInfo = inlineFlag)
```

Comment:
Should this be removed?

Reply:
The bugfix for this has been merged, so it can. Will do.